robottelo/ui/my_account.py

```diff
@@ -10,3 +11,48 @@ class MyAccount(Base):
     def navigate_to_entity(self):
         """Navigate to My Account page"""
         Navigator(self.browser).go_to_my_account()
+
+    def _update_helper(self, input_locator_name, new_value,
```

Comment:
I saw that on org update as well but have to disagree it is user friendly. At least on the code we are mixing several responsibilities and that is why there is the need for such huge number of conditionals. I am going to think in some middle ground here.